marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -183,6 +183,11 @@ class Jobs {
     if (batchStatus === "finished" || batchStatus === "finished_with_errors" || batchStatus === "failed") {
       docObj.batch.timeEnded = fn.currentDateTime();
     }
+    if(stackTrace != null){
```

Comment:
You can rely on the truthy/falsey characteristics of JavaScript (https://j11y.io/javascript/truthy-falsey/) anf the variable name error is more clear.

```suggestion
    if(error){
```